cli-test/spec/test_cases/add/tests.rb

```
@@ -1,52 +1,56 @@
     aruba.config.exit_timeout = 120
   end
 
   after(:each) do
-    run_command_and_stop('rm -rf test-relative-paths')
+    FileUtils.rm_rf('test-relative-paths')
   end
 
   it 'tests oxen add with relative paths from subdirectories' do
-    directory_path = 'tmp/aruba/test-relative-paths'
-
+    directory_path = "tmp\\aruba\\test-relative-paths"
```

RuboCop (1.69.1)

[warning] Useless assignment to variable - `directory_path`.

(Lint/UselessAssignment)